src/index.html

```diff
@@ -4,19 +4,61 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0"
     />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
+    <title>Moyo Header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
     />
-    <title>Moyo header</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="main-header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo Logo"
+          width="40"
+          height="40"
+          loading="lazy"
+        />
+      </a>
+      <nav
+        class="main-menu"
+        aria-label="Main navigation"
+      >
+        <ul class="main-menu__list">
+          <li class="main-menu__item">
+            <a
+              href="#"
+              class="is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="main-menu__item"><a href="#">Samsung</a></li>
```

Comment:
```suggestion
          <li class="main-menu__item"><a class="main-menu__link" href="#">Samsung</a></li>
```
Consider using classes for styling links